Tests/conf.json

```diff
@@ -1853,6 +1853,11 @@
         {
             "playbookID": "Request Debugging - Test",
             "fromversion": "5.0.0"
+        },
+                {
```

Comment:
please fix indentation here. it's not an issue but looks bad

Reply:
Fixed thanks